app/mailers/spree/order_mailer_decorator.rb

```diff
@@ -2,7 +2,7 @@
   def gift_card_email(card_id, order_id)
     @gift_card = Spree::GiftCard.find(card_id)
     @order = Spree::Order.find(order_id)
-    subject = "#{Spree::Config[:site_name]} Gift Card"
+    subject = "Gift Card"
```

Comment:
@AnuragBhardwaj1 Please refer Spree 3.0.4 for subject. There's a common pattern that is followed for subject and use the same.